docs/basics.md

```diff
@@ -254,3 +255,35 @@ In this example, `model` and `something` are directly available on the
 `MyObject` instance, while `another` must be accessed via `getOption`. This is
 handy when you want to add extra keys that will be used heavily throughout the
 defined class.
+
+### The `options` Property
+
+Is possible to define an `options` property in the class definition that will
+be used as default values. The `options` argument passed at class instantiation
+has precedence over the property.
```

Comment:
Reading the code example below, I'm taking this to mean:

> Options passed in will be merged with this `options` property.
> The arguments passed in at instantiation will override properties set on the `options` attribute.

Reply:
Updated